### ISSUE TITLE
seeds.rb not creating page parts correctly

### ISSUE THREAD

**Issue by RichardM90** [refinery/refinerycms-calendar#55]
refinerycms v3.0.6
refinery-calendar v3.0.0

seeds.rb has code to generate page parts for events and venues but this results in refinery_page_parts with values for slug = 'title_body_slug_body' and title = '{:title=>"Body", :slug=>"body"}'

I think this code ...
`  Refinery::Pages.default_parts.each_with_index do |default_page_part, index|
    page.parts.create(:title => default_page_part, :body => nil, :position => index)
  end`

should be ...
`  Refinery::Pages.default_parts.each_with_index do |default_page_part, index|
    page.parts.create(:title => default_page_part[:title], :slug => default_page_part[:slug], :body => nil, :position => index)
  end`

**Comment by bricesanchez** [comment]
Hi @RichardM90! thanks for your bug report! Could you provide a bugfix? :)

**Comment by RichardM90** [comment]
Absolutely, more than happy to. This will be my first contribution to a public open source project. Is the following guide the correct flow I should follow?
https://git-scm.com/book/en/v2/GitHub-Contributing-to-a-Project

**Comment by bricesanchez** [comment]
You could read this too: https://github.com/refinery/refinerycms/blob/master/contributing.md
